Libraries/LibCore/DirIterator.cpp

```diff
@@ -108,8 +108,8 @@ String find_executable_in_path(String filename)
         return {};
     }
 
-    for (auto directory : StringView { getenv("PATH") }.split_view(':')) {
-        auto fullpath = String::format("%s/%s", directory, filename);
+    for (auto directory : String { getenv("PATH") }.split(':')) {
+        auto fullpath = String::format("%s/%s", directory.characters(), filename.characters());
```

Comment:
FYI you can also format() a StringView by doing `"%.*s", directory.length(), directory.characters_without_null_termination()`

Reply:
I actually knew that, but it's soo cumbersome.